nx-X11/lib/X11/Imakefile

```diff
@@ -945,7 +941,6 @@ LCSRCS = \
 	lcConv.c \
 	lcDB.c \
 	lcDefConv.c \
-	lcDynamic.c \
```

Comment:
`lcDynamic.[co]` also become obsolete, so remove them from the source tree as well and add them to commit message.